docs/guides/query-kyc-state-on-c-chain.md

```diff
@@ -0,0 +1,164 @@
+<figcaption align="center"><b>Fig.5:</b> Click "Deploy and run transactions" tab and select "Inject Provider" from "Environment"</figcaption>
+</figure>
+
+- Verify that the MetaMask is connected to the Remix IDE.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/5-check-network-connected.png"/>
+<figcaption align="center"><b>Fig.6:</b> Verify network is connected</figcaption>
+</figure>
+
+## Downloading the CaminoAdmin.abi file
+
+We can either compile the contract or use the precompiled ABI file. In this guide, we will use the ABI file.
+
+You can download the ABI file from Chain4Travel's GitHub repository:
+[chain4travel/caminoethvm - CaminoAdmin.abi](https://github.com/chain4travel/caminoethvm/blob/chain4travel/contracts/build_contracts/admin/abi/CaminoAdmin.abi)
+
+- Rigth click on "Raw" and select "Save as..." to save it to your local computer.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/6-abi-save-link-as.png"/>
+<figcaption align="center"><b>Fig.7:</b> Save "CaminoAdmin.abi" file to local computer</figcaption>
+</figure>
+
+## Uploading the ABI file to the Remix IDE
+
+Follow these steps to upload the ABI file to the Remix IDE:
+
+- Switch to **File Explorer** on the right sidebar and click the upload button.
+- Select the file you downloaded from your local computer and upload it.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/7-load-local-file.png"/>
+<figcaption align="center"><b>Fig.8:</b> Switch to "File Explorer" and click "Load a local file into current workspace"</figcaption>
+</figure>
+
+- Select the `CaminoAdmin.abi` file.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/8-select-the-loaded-file.png"/>
+<figcaption align="center"><b>Fig.9:</b> Select the loaded ABI file</figcaption>
+</figure>
+
+## Getting the CaminoAdmin Contract Address
+
+**Camino Admin Address**: `0x010000000000000000000000000000000000000a`
+
+You can also find the Camino contract addresses on [Camino Smart Contracts](/developer/references/camino-contracts.md) page under `Developer` > `References`.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/9-get-the-contract-address.png"/>
+<figcaption align="center"><b>Fig.10:</b> Get the CaminoAdmin contract address</figcaption>
+</figure>
+
+## Interacting with the Address
+
+- Paste the address into the **At Address** field and click on the button.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/10-paste-address-and-click-at-address.png"/>
+<figcaption align="center"><b>Fig.11:</b> Paste the contract address into the "At Address" field and click "At Address"</figcaption>
+</figure>
+
+- Confirm the interaction
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/11-confirm-the-popup.png"/>
+<figcaption align="center"><b>Fig.12:</b> Confirm the dialog popup</figcaption>
+</figure>
+
+## Making the Query
+
+Follow the steps below to make the query:
```

Comment:
This is the manual query of course, it might be useful to specify how this can be written into the smart contract, without necessarily giving the step-by-step, but at least being very clear that the check can be automated.